ha/util/conf_store.py

```diff
@@ -110,3 +121,96 @@ def set_cluster_cardinality(self, index):
         cluster_cardinality_key = CLUSTER_CARDINALITY_KEY
         cluster_cardinality_value = {CLUSTER_CARDINALITY_NUM_NODES: num_pods, CLUSTER_CARDINALITY_LIST_NODES : watch_pods }
         self._confstore.update(cluster_cardinality_key, json.dumps(cluster_cardinality_value))
+
+    @staticmethod
+    def get_cvg_list(index, node_id):
```

Comment:
There are many functionalities common across these 3 functions.
See if you can create a separate functions for the common functionalities  and use them in these 3 functions

Reply:
Done.